src/abstractdataframe/selection.jl

```diff
@@ -287,6 +333,17 @@ julia> df
 │     │ Int64 │ Int64 │
 ├─────┼───────┼───────┤
 │ 1   │ 6     │ 15    │
+
+julia> df = DataFrame(a=1:3, b=4:6);
+
+julia> select!(df, AsTable(:) => ByRow(x -> map(x -> x^2, x)))
```

Comment:
Maybe also show an example which is useful in practice, like the row means?

Reply:
ok - changed